README.md

```diff
@@ -166,7 +166,7 @@ stringData:
 
 ## Considerations for Role-Based Access Control (RBAC)
 
-Cluster operators and CRD authors **SHOULD** opt-in resources to expose provisioned services by defining a `ClusterRole` with a label matching `servicebinding.io/controller=true`, the `get`, `list`, and `watch` verbs **MUST** be granted.
+Cluster operators and CRD authors **SHOULD** opt-in resources to expose provisioned services by defining a `ClusterRole` with a label matching `servicebinding.io/controller=true`.  The `get`, `list`, and `watch` verbs **MUST** be granted.
```

Comment:
The second sentence feels a bit disconnected from the previous one. How about mentioning the `ClusterRole` once again in this sentence?
```suggestion
Cluster operators and CRD authors **SHOULD** opt-in resources to expose provisioned services by defining a `ClusterRole` with a label matching `servicebinding.io/controller=true`.  The `get`, `list`, and `watch` verbs **MUST** be granted in the `ClusterRole`.
```
If this looks good, please make similar changes in the other RBAC section also.

Reply:
I agree, it looks better when the `ClusterRole` is mentioned again.  Fixed.